src/emsarray/conventions/_fixes.py

```diff
@@ -0,0 +1,70 @@
+import abc
```

Comment:
TODO: Add tests for this entire shenanigan

Reply:
This entire shenanigan will be moved to a new pull request. It is too much on top of this change.